test/gurka/gurka.h

```diff
@@ -560,31 +560,36 @@ std::tuple<const baldr::GraphId,
            const baldr::DirectedEdge*>
 findEdge(valhalla::baldr::GraphReader& reader,
          const std::unordered_map<std::string, midgard::PointLL>& nodes,
-         const baldr::GraphId& tile_id,
          const std::string& way_name,
-         const std::string& end_node) {
-  auto* tile = reader.GetGraphTile(tile_id);
+         const std::string& end_node,
+         const baldr::GraphId& tile_id = baldr::GraphId{}) {
+  // if the tile was specified use it otherwise scan everything
```

Comment:
i made it so you didnt have to specify the tile id if you didnt want to. for most tests the tilesets are so small we just let the function search over all edges in the graph. if you have a massive data set you can still specify the tile to speed it up